### PR TITLE
adds Flipcard BB

### PR DESCRIPTION
**Pull request by rafibomb**

### CODE REVIEW

**Comment by rafibomb** [comment]
@abdullahsalem Would you mind looking this over for merge?

**Comment by IamManchanda** [comment]
@rafibomb @kball :disappointed: 

Does this mean that hopes of https://github.com/zurb/foundation-sites/issues/9551 coming into the framework core/motion ui/etc is dead and has come to an end??

File: src/building-blocks/flip-card/flip-card.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+$flip-card-transition-speed: 0.6s;
+
+.flip-card {
+  perspective: 1000;
+  border: 0;
+  background: transparent;
+}
+
+.flip-card:hover .flip-card-inner,
+.flip-card.hover .flip-card-inner {
```
Comment (IamManchanda) [inline src/building-blocks/flip-card/flip-card.scss]: `&:hover` and `&.hover` should be used here !

Example => http://codepen.io/IamManchanda/pen/eBoYpM/

> ```scss
> .card-flip {
>   perspective: 1000px;
>
>   &:hover .flip,
>   &.hover .flip {
>     transform: rotateY(180deg);
>   }
> }
> ```

**Comment by rafibomb** [comment]
MotionUi can incorporate an animation like this - especially useful when coupled with some JS or a touch library.

**Comment by IamManchanda** [comment]
So @rafibomb what's the call you taking?

**Comment by rafibomb** [comment]
In think that animation mixins are great and adding something like this to MotionUI makes sense to me.

File: src/building-blocks/flip-card/flip-card.scss (hunk 1 above)
Comment (rafibomb) [inline src/building-blocks/flip-card/flip-card.scss]: I don't think this matters - not more readable - not saving much code - but I did it anyways :)

**Comment by IamManchanda** [comment]
> I don't think this matters - not more readable - not saving much code - but I did it anyways :)

@rafibomb But that's the way sass is written... Just being purist!

**Pull request merged**